x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/action/huggingface/HuggingFaceAction.java

```diff
@@ -62,17 +61,16 @@ public HuggingFaceAction(
             serviceComponents.threadPool()
         );
         this.account = new HuggingFaceAccount(model.getUri(), model.getApiKey());
-        this.errorMessage = format("Failed to send Hugging Face %s request to [%s]", requestType, model.getUri().toString());
+        this.errorMessage = format("Failed to send Hugging Face %s request from model id [%s]", requestType, model.getModelId());
```

Comment:
is this an inferenceEntityId or a modelVariantID?

Reply:
`inferenceEntityId`